metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -600,15 +642,24 @@
   [context concept-type concept-id revision-id]
   ;; delete the related tag associations and variable associations
   (delete-associated-tag-associations context concept-id revision-id)
-  (delete-associated-variable-associations context concept-type concept-id revision-id)
+  (delete-associations context concept-type concept-id revision-id :variable-association)
+  (delete-associations context concept-type concept-id revision-id :service-association)
   (ingest-events/publish-concept-revision-delete-msg context concept-id revision-id))
 
 (defmethod force-delete-cascading-events :variable
   [context concept-type concept-id revision-id]
   ;; delete the related variable associations
-  (delete-associated-variable-associations context concept-type concept-id revision-id)
+  (delete-associations context concept-type concept-id revision-id :variable-association)
   (ingest-events/publish-concept-revision-delete-msg context concept-id revision-id))
 
+(defmethod force-delete-cascading-events :service
+  [context concept-type concept-id revision-id]
+  ;; delete the related service associations
+  (delete-associations context concept-type concept-id revision-id :service-association)
+  ;; Do not queue the concept revision deletion event for now.
+  ; (ingest-events/publish-concept-revision-delete-msg context concept-id revision-id)
```

Comment:
Add a ticket number

Reply:
We have not written any ticket or decided if we want to support all revisions search for services.